Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -379,9 +330,28 @@ namespace PhysX
         }
     }
 
+    bool EditorProxyShapeConfig::IsNonUniformlyScaledPrimitive() const
+    {
+        return m_hasNonUniformScale && (Utils::IsPrimitiveShape(GetCurrent()) || IsCylinderConfig());
```

Comment:
nit: Second parens around `Utils::IsPrimitiveShape` are redundant and I feel make this a little harder to read as it looks like the brackets might be around the final condition

Reply:
There is no second parenthesis around `Utils::IsPrimitiveShape`.

Reply:
Apologies I misread this, ignore me 🤦